frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -3,73 +3,85 @@
 import ReplyIcon from '@material-ui/icons/Reply';
 // import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
-import ModalPopup from '../controllers/ModalPopup';
+import ModalPopup from './ModalPopup';
 import { getcurrentStateKey, getMultiPopupOpenState, setMultiPopupOpen } from '../app/Service';
 import {
   getSmoothedSpO2,
   getSmoothedHR,
-  roundValue,
   getParametersRequestDraftFiO2,
   getParametersRequestDraftFlow,
+  getSpO2AlarmLimitsRequest,
+  getHRAlarmLimitsRequest,
+  getParametersFlow,
+  getParametersFiO2,
 } from '../../store/controller/selectors';
 import { SetValueContent } from '../controllers/ValueModal';
 import { a11yProps, TabPanel } from '../controllers/TabPanel';
 import ValueInfo from '../dashboard/components/ValueInfo';
 import { BPM, LMIN, PERCENT } from '../info/units';
 import { AlarmModal } from '../controllers';
-import { ParametersRequest, AlarmLimitsRequest } from '../../store/proto/mcu_pb';
+import { ParametersRequest, AlarmLimitsRequest, Range } from '../../store/proto/mcu_pb';
 import { MessageType } from '../../store/proto/types';
 import { commitRequest, commitDraftRequest } from '../../store/controller/actions';
-import store from '../../store';
 
 /**
- * @typedef Data
+ * @typedef InternalState
  *
- * Interface for Data
+ * Interface for InternalState
  *
  * @prop {string} stateKey Unique identifier for the value
  * @prop {string} label Label for the value
  * @prop {string} units Unit measurement of the value
- * @prop {boolean} isAlarmEnabled Config to set if value is Alarm type
- * @prop {boolean} isSetvalEnabled Config to set if value is Set type
+ * @prop {boolean} isAlarmLimitsEnabled Config to set if value is Alarm type
+ * @prop {boolean} isParamEnabled Config to set if value is Set type
  * @prop {number | null} committedSetting Current value
- * @prop {number[]} alarmValues Alarm range values
- * @prop {number} setValue Set Value
+ * @prop {Range | null} alarmLimitsDraft Alarm range values
+ * @prop {number} paramDraft Set Value
  * @prop {number | null} minValue Minimum under which value cannot decrement
  * @prop {number | null} maxValue Maximum above which value cannot increment
  * @prop {number | null} alarmLimitMin Alarm limit Minimum Range value
  * @prop {number | null} alarmLimitMax Alarm limit Maximum Range value
- * @prop {number[]} alarmValuesActual Actual Alarm Value when Component was initalized
- * @prop {number} setValueActual Actual Set Value when Component was initalized
+ * @prop {Range | null} alarmLimitsCurrent Actual Alarm Value when Component was initalized
+ * @prop {number} paramCurrent Actual Set Value when Component was initalized
  */
-interface Data {
+interface InternalState {
   stateKey: string;
   label: string;
   units: string;
-  isAlarmEnabled: boolean;
-  isSetvalEnabled: boolean;
+  isAlarmLimitsEnabled: boolean;
+  isParamEnabled: boolean;
   committedSetting?: number | null;
```

Comment:
What does `committedSetting` describe? Is it `committedParam`, or does it also apply to alarm limits?

Reply:
To my understanding, it's the DraftParam value we pass to the child component `SetValueContent`  which is the modal to set `FiO2` and `Flow`, see [here](https://github.com/pez-globo/pufferfish-software/blob/develop/frontend/src/modules/controllers/ValueModal.tsx#L100), so No it does not apply to alarm limits, so yes maybe I'll change it to `committedParam`